dds_web/config.py

```diff
@@ -25,15 +25,12 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
     # Data related config
-    MAX_CONTENT_LENGTH = 16777216
+    MAX_CONTENT_LENGTH = 0x1000000
     MAX_DOWNLOAD_LIMIT = 1000000000
 
     # Expected paths - these are the bind paths *inside* the container
     USE_LOCAL_DB = True
     LOGS_DIR = "/dds_web/logs"
-    UPLOAD_FOLDER = "/dds_web/uploads"
-    DOWNLOAD_FOLDER = "/dds_web/downloads"
-    LOCAL_TEMP_CACHE = "/dds_web/local_temp_cache"
     DDS_S3_CONFIG = "/code/dds_web/sensitive/s3_config.json"
```

Comment:
@inaod568 we can remove the `DDS_S3_CONFIG` as well right?

Reply:
That should mean that I can remove `s3_config.json? as well from the comment in `docker_compose.yml`?

Reply:
Yes it would. I think we simply moved the values from that one into the general config file.

Reply:
Yes 👍🏻